### PR TITLE
[csharp/en] Update C Sharp markdown.

### PR DESCRIPTION
**Pull request by StarlkYT**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]` (example `[python/fr-fr]` or `[java/en]`)
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: csharp.html.markdown [hunk 1]
```diff
@@ -48,6 +48,10 @@ using System.Data.Entity;
 
 // Namespaces define scope to organize code into "packages" or "modules"
 // Using this code from another source file: using Learning.CSharp;
+
+// You can also do this in C# 10, it is called file-scoped namespaces.
+// namespace Learning.CSharp;
```
Comment (Menelion) [inline csharp.html.markdown:53]: Oh, something is taken from PHP into C# and not the other way around 😊

File: csharp.html.markdown [hunk 2]
```diff
@@ -1299,16 +1315,27 @@ namespace Csharp7
...
- 	* ASP.NET Web Forms (old)
- 	* WebMatrix (tool)
+    * ASP.NET Core ✨
+
  * Desktop Development
- 	* Windows Presentation Foundation (WPF) (new)
- 	* Winforms (old)
+ 	* Windows Presentation Foundation 👍 🎈 🎁
+    * Universal Windows Platform ✨ 🎁
+    * Uno Platform 🔥 ✨
```
Comment (Menelion) [inline csharp.html.markdown:1330]: Is it that new thing that supersedes UWP?

**Review by Menelion — APPROVED** [review]
Thank you so much! If you could cover at least one or couple topics marked uncovered, it would be super-mega-nice 😊

File: csharp.html.markdown (hunk 2 above)
Comment (StarlkYT) [reply csharp.html.markdown:1330]: I wouldn't say supersedes UWP. You can use WinUI 2 controls inside UWP or WPF/WinForms (Via XAML Islands) too

**Pull request merged**